tests/excerptexport/test_downloads.py

```diff
@@ -24,7 +24,8 @@ def test_file_download(self, *args):
         excerpt = Excerpt.objects.create(name='Neverland', is_active=True, is_public=True, owner=user,
                                          bounding_geometry=bg)
         extraction_order = ExtractionOrder.objects.create(excerpt=excerpt, orderer=user)
-        output_file = OutputFile.objects.create(mime_type='test/plain', extraction_order=extraction_order,
+        export = extraction_order.exports.create(file_format='fgdb')
+        output_file = OutputFile.objects.create(mime_type='test/plain', export=export,
```

Comment:
The result files are zipped, right? I wasn't able to determine the type of format by the file because our zips are of a custom structure and containing directories and somwhere in there some geo format. So the easiest way to do this was to hard code the mime type in the process receiving the geo file from the converter. Because at this point, we did know which format is inside the zip. But I'm not shure if this problem already exist about the new architecture.